modules/openapi-generator/src/main/java/org/openapitools/codegen/languages/Swift5ClientCodegen.java

```diff
@@ -499,7 +504,7 @@ public String escapeReservedWord(String name) {
         if (this.reservedWordsMappings().containsKey(name)) {
             return this.reservedWordsMappings().get(name);
         }
-        return "_" + name;  // add an underscore to the name
```

Comment:
Could you explain this change please?

Reply:
Swift uses backticks to escape reserved words. Refer to the note section [here](https://docs.swift.org/swift-book/LanguageGuide/TheBasics.html#ID313)

Reply:
This is a breaking change, that I suggested to open a new PR with, against the `6.0.x` branch

Reply:
I noticed that this breaks `objCompatible` files, so I changed it to escape `objCompatible` reserved words with a `_` and back-ticks otherwise.